src/pages/Call/components/transcript-call/transcript-call.tsx

```diff
@@ -0,0 +1,58 @@
+import "./transcript-call.scss";
+
+import { IonButton, IonIcon } from "@ionic/react";
+import { volumeMediumOutline, micOutline, sendOutline } from "ionicons/icons";
+
+import { IonCol, IonContent, IonHeader, IonPage, IonRow } from "@ionic/react";
+
+const TranscriptCall = () => {
+  const transcript = [
+    {
```

Comment:
The websocket connection with the backend is ready. It is available with the `useChatWebSocket` hook. We can start adding the real texts for this